insonmnia/dwh/server.go

```diff
@@ -998,6 +999,7 @@ func (w *DWH) onDealOpened(dealID *big.Int) error {
 
 	_, err = tx.Exec(
 		w.commands["insertDealCondition"],
+		nil,
```

Comment:
Calling methods like `doSomething(nil, nil, nil, nil, ...)` clearly don't improve readability, so I for example have no idea what's going on here.

Reply:
this will go away with the Postgres PR, it was an intermediate step.